client/my-sites/google-my-business/google-my-business-stats/index.js

```diff
@@ -0,0 +1,315 @@
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="time" size={ 18 } /> 9:00 AM - 6:00 PM
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="globe" size={ 18 } /> www.catescookies.com
+						</div>
+						<div className="google-my-business-stats__address">
+							<Gridicon icon="location" size={ 18 } />
+							<div className="google-my-business-stats__listing-detail">
+								345 North Avenue<br />Talihassee, FL 34342<br />USA
+							</div>
+						</div>
+						<div className="google-my-business-stats__listing-detail">
+							<Gridicon icon="tag" size={ 18 } /> Bakery
+						</div>
+						<Button
+							className="google-my-business-stats__listing-button"
+							href="https://business.google.com"
+							target="_blank"
+						>
+							Update Listing
+							<Gridicon icon="external" />
+						</Button>
+					</Card>
+
+					<SectionHeader label={ translate( 'How customers search for your business' ) } />
+					<Card>
+						<img src="/calypso/images/google-my-business/pie-chart.svg" />
+						<div className="google-my-business-stats__stat-title">451 Total searches</div>
+						<div className="google-my-business-stats__search-type">
+							<span className="google-my-business-stats__search-type-direct" />
+							Direct<br />
+							362 (60%)<br />
+							<span className="google-my-business-stats__search-type-description">
+								Customers who find your listing searching for your business name or address
+							</span>
+						</div>
+
+						<div className="google-my-business-stats__search-type">
+							<span className="google-my-business-stats__search-type-discovery" />
+							Discovery<br />
+							89 (40%)<br />
+							<span className="google-my-business-stats__search-type-description">
+								Customers who find your listing searching for a category, product, or service
+							</span>
+						</div>
+					</Card>
+
+					<Card>
+						<div className="google-my-business-stats__feature">
+							<p>
+								Businesses with recent photos typically receive more clicks to their business
+								websites.
+							</p>
+							<img
+								src="/calypso/images/google-my-business/reviews.svg"
+								className="google-my-business-stats__feature-image"
+							/>
+						</div>
+						<Button primary className="google-my-business-stats__feature-button">
+							Post Photos
+							<Gridicon icon="external" />
+						</Button>
+					</Card>
+
+					<SectionHeader
+						label={ translate( 'Where your customers view your business on Google' ) }
+					/>
+					<Card>
+						<p className="google-my-business-stats__description">
+							The Google services that customers use to find your business
+						</p>
+						<FormFieldset>
+							<FormSelect className="google-my-business-stats__select">
+								<option>{ translate( '1 week' ) }</option>
+								<option>{ translate( '1 month' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'month' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gmb-prototype)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Or even better `translate( '%s month', '%s months', { count: 1, args: '1' } );`.